market/uipath/meta.json

```diff
@@ -8,7 +8,9 @@
 		"id": "open-api-rest-client",
 		"properties": {
 			"openApiUrl": "https://platform.uipath.com/AXONPRESALES/AXONPRESALES/swagger/v11.0/swagger.json",
-			"namespace": "com.uipath.orchestrator"
+			"namespace": "com.uipath.orchestrator",
+			"features": "com.axonivy.connectivity.rest.client.auth.OAuth2Feature,com.axonivy.connectivity.rest.sample.uipath.TenantHeaderFeature,com.axonivy.connectivity.rest.json.OpenApiJsonFeature",
+			"properties": "jersey.config.client.followRedirects=false,tenant=AXONPRESALES"
```

Comment:
In the future we should opt for a format here, that makes it clear how 'lists' can be provide. Now we have a comma separator within the string - but that's not common json.